static/linux/postrm

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+set -e
+rm /usr/bin/webtorrent-desktop
```

Comment:
everything under `/opt/webtorrent-desktop` its handled by the package manager ( directory and files creation/removal)

`/usr/bin/webtorrent-desktop` is a symbolic link to `/opt/webtorrent-desktop/WebTorrent` created post-installation (check linux/postinst file) so it should be removed post-package removal (linux/postrm)